exercises/diffie-hellman/diffie_hellman_test.exs

```diff
@@ -0,0 +1,98 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("diffie_hellman.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule DiffieHellmanTest do
+  use ExUnit.Case
+
+  #@tag :pending
+  test "private key should be random between 1 and P-1, inclusive" do
+    prime_p = 23
+
+    generated_private_keys = 1..100 |> Enum.map(fn _i -> DiffieHellman.generate_private_key(prime_p) end)
```

Comment:
Cold this be moved to a function on `DiffieHellman` and driven out with a test? If you don't agree with that, then I suggest extracting it to a helper function.